nixos/doc/manual/manual-contrib/manual-contrib.xml

```diff
@@ -0,0 +1,13 @@
+<appendix xmlns="http://docbook.org/ns/docbook"
+          xmlns:xlink="http://www.w3.org/1999/xlink"
+          xmlns:xi="http://www.w3.org/2001/XInclude"
+          version="5.0"
+          xml:id="ch-manual-contrib">
+ <title>Contributing to this manual</title>
+ <para>
+  The NixOS manual is maintained as part of the
+  <link xlink:href="https://github.com/nixos/nixpkgs">nixpkgs</link> collection.
+  You can find the sources and documentation on how to build them in
+  the <link xlink:href="https://github.com/nixos/nixpkgs/tree/master/nixos/doc/manual">nixos/doc/manual</link> directory.
```

Comment:
Maybe instead of this, we should transfer the not too long contents of the README from there to here, and put in the README instead:

```
Transferred to: <URL>
```

?

It should save future contributors a few clicks.

Reply:
I think it's nice to keep it in the README - to make a contribution you'll have to check out the nixpkgs collection anyway, and once you know where to find it having the exact command right there instead of having to look at the docs themselves is more convenient.

Reply:
I understand the argument. I think we should make it inline with:

https://nixos.org/manual/nixpkgs/stable/#chap-contributing